### DISCUSSION TITLE
Can't find parameter['teffratio', 'requivsumfrac'] in default_binary()

### DISCUSSION THREAD

**Discussion by d11122** [phoebe-project/phoebe2#499] [Questions]
I can't find parameter['teffratio', 'requivsumfrac']  in the default _binary(). 
when I create a new default _binary(), I Tried `print(b['teffratio'])`, 
got 
`ParameterSet: 0 parameters
NO PARAMETERS`
But, I know that after using ebai, I can get these parameter.
Do I need to follow the tutorial to create these parameters myself without using ebai? --daimin, thanks.

**Comment by kecnry** [comment]
These are both optional constraints (they don't exist by default, but are added for you with the ebai solver since it requires them).  You can add them manually with [b.add_constraint](http://phoebe-project.org/docs/latest/api/phoebe.frontend.bundle.Bundle.add_constraint):

```
import phoebe

b = phoebe.default_binary()
b.add_constraint('teffratio')
b.add_constraint('requivsumfrac')
```

**Comment by d11122** [comment]
thanks you! Kyle Conroy
